### PR TITLE
[WIP] second version GiantSteps key

### PR DESCRIPTION
**Pull request by PRamoneda**
#307

### CODE REVIEW

File: mirdata/giantsteps_key.py [hunk 1]
```diff
@@ -256,10 +275,23 @@ def load_key(keys_path):
...
+        'pitch-description': new_annotations['pitch-description'],
+        'pc-set': new_annotations['pc-set'],
+        'cardinality': cardinality,
+        'start_times': new_annotations['start_times'],
+        'end_times': new_annotations['end_times'],
+        'comments': new_annotations['comments'],
+    }
 
 
 def load_tempo(metadata_path):
```
Comment (genisplaja) [inline mirdata/giantsteps_key.py:297]: Maybe you don't need to create functions to load tempo if you can retrieve it from `metadata` in the `__init__` of Track. I think it's the same for `genres` and `artists`.
You would load the metadata with `utils.LargeData` and then: `self.genre = self._track_metadata['bpm']`. I think that would be easier and you wouldn't need to create all these catched properties and all these methods. 
Am I correct?

File: mirdata/giantsteps_key.py [hunk 2]
```diff
@@ -37,18 +37,18 @@
 
 DATASET_DIR = 'GiantSteps_key'
 REMOTES = {
+    'new_annotations': download_utils.RemoteFileMetadata(
```
Comment (rabitt) [inline mirdata/giantsteps_key.py:40]: is there a new publication for the new annotations as well?

**Review by nkundiushuti — APPROVED** [review]

File: scripts/make_giantsteps_key.py [hunk 3]
```diff
@@ -29,14 +29,14 @@ def md5(file_path):
 def make_giantsteps_key_index(data_path):
     meta_dir = os.path.join(data_path, 'meta')
     audio_dir = os.path.join(data_path, 'audio')
-    key_dir = os.path.join(data_path, 'keys_gs+')
+    key_dir = os.path.join(data_path, 'new_annotations')
```
Comment (rabitt) [inline scripts/make_giantsteps_key.py:32]: nit: can we give this a more descriptive name, like annoations_version2?

**Review by rabitt — APPROVED** [review]
👍

**Comment by magdalenafuentes** [comment]
I see this PR has been approved already, @PRamoneda what are we missing in this loader to be able to merge? (besides solving the conflicts)

**Comment by PRamoneda** [comment]
I have found some errors in Giantsteps Key and I have to speak with Angel faraldo

**Comment by PRamoneda** [comment]
Hi! We have to fix some things in the dataset and we are going to a new version. Then this PR is stopped until that!

**Comment by PRamoneda** [comment]
@magdalenafuentes @genisplaja  you can close this thread. We are not going to do a new version

**Comment by guillemcortes** [comment]
Closing the PR as per @PRamoneda's comment.

**Pull request closed without merging**